lib/rack/throttle/rules.rb

```diff
@@ -46,14 +46,11 @@ def rule_whitelisted?(request)
       end
 
       def rule_for(request)
-        cache.fetch("rule-#{request.object_id}") do 
```

Comment:
so... no more caching of the rules?

Reply:
procs can't be marshaled into something cache-able, and only caching the index if the rule in the array seems somehow hacky.